### PR TITLE
Add frogr stable

### PR DESCRIPTION
**Pull request by mariospr**

### CODE REVIEW

File: org.gnome.frogr.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "sdk": "org.gnome.Sdk",
+    "command": "frogr",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Needed for frogr to access your pictures */
```
Comment (TingPing) [inline org.gnome.frogr.json]: `--filesystem=xdg-pictures` perhaps?

File: org.gnome.frogr.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": ["/include", "/lib/pkgconfig",
```
Comment (TingPing) [inline org.gnome.frogr.json]: Seems like this cleanup line was just copy pasted and has leftovers?

File: org.gnome.frogr.json (hunk 1 above)
Comment (mariospr) [inline org.gnome.frogr.json]: I've tried something like that a while ago, and I even had `--filesystem=home` for a while, but that was very inconvenient in common use cases that I'd like frogr to support, such as loading pictures from an external drive (or camera) or from a remote server (e.g. NAS).

I understand, though, that `host` is a much wider permission than any of those, so I'm open to change it if that makes sense for flathub (after all, people can override those with `flatpak override`)

What do you think?

File: org.gnome.frogr.json (hunk 2 above)
Comment (mariospr) [inline org.gnome.frogr.json]: Err... yes. It's pretty much "generic copy & paste" here, no need to cleanup header or .pc files that don't exist anyway. I'll fix that.

**Comment by mariospr** [comment]
@TingPing I've updated the PR with a change for the cleanup rule, but haven't changed the --filesystem permission for now. Any comment?

**Comment by TingPing** [comment]
If the application is functional for a majority of users with `xdg-pictures` then I think we should do that since as you mentioned extra directories can be added manually. Hopefully we get integration in Gnome-Software or such to add overrides at some point.

**Comment by mariospr** [comment]
> If the application is functional for a majority of users with xdg-pictures then I think we should do that since as you mentioned extra directories can be added manually. Hopefully we get integration in Gnome-Software or such to add overrides at some point.

Fair enough, that makes sense to me too. I've just updated my branch to include xdg-pictures, xdg-videos (frogr also loads videos) and xdg-documents (for saving/loading project files).

Let me know what you think, thanks!

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/5/builds/422

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/org.gnome.frogr

**Comment by mariospr** [comment]
Thanks!

**Pull request closed without merging**